state_machine/applications/system-check/main.py

```diff
@@ -30,7 +31,8 @@
 all_tests = [
     ("SD Test", "sd", tests.sd_test, True),
     ("IMU Test", "imu", tests.imu_test, True),
-    ("Sun Sensor Test", "sun", tests.sun_sensor_test, True),
+    ("tsl2561 Sun Sensor Test", "sun", tests.sun_sensor_test, True),
```

Comment:
It would be nice to have this set so someone who doesn't know which sensors are present can run it. Maybe check the hardware configuration in the sun_sensor_test call rather than having two separate options here?

Reply:
I think if you just make the interface the same for both sun sensor drivers, you won't actually have to change the system check stuff at all (except to add your consistency check below).